espnet2/tasks/enh.py

```diff
@@ -328,6 +328,28 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             help="The set of all possible categories in the dataset. Used to add the "
             "category information to each sample",
         )
+        group.add_argument(
+            "--speech_segment",
+            type=int_or_none,
+            default=None,
+            help="Truncate the audios to the specified length if not None",
+        )
+        group.add_argument(
+            "--avoid_allzero_segment",
```

Comment:
Is it necessary to provide this option? I feel the default `True` is fine.

Reply:
Sometimes we may want to train the model to generate silence, and sometimes we may want to avoid the silence as the training target depending on the loss function we use. So I just make this argument to enable both possibilities.